### PR TITLE
Feature/autodelete old targets

### PR DESCRIPTION
**Pull request by JorgeMichelena**

### CODE REVIEW

File: target/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import, unicode_literals
+from .tasks import app as celery_app
+
+__all__ = ('celery_app',)
```
Comment (kaozdl) [inline target/__init__.py:4]: Missing newline

File: targets/validators.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ def validate_coordinates(value):
         )
 
 def less_than_10_targets(user):
-    if user.targets.count>=10:
+    if user.targets.count()>=10:
```
Comment (kaozdl) [inline targets/validators.py:19]:
```suggestion
    if user.targets.count() >= 10:
```

File: target/tasks.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
```
Comment (kaozdl) [inline target/tasks.py:1]: future is the backports library from python 2 why are you using it here?

File: target/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import, unicode_literals
```
Comment (kaozdl) [inline target/__init__.py:1]: you don't need this in python3

File: target/settings.py [hunk 5]
```diff
@@ -174,3 +175,17 @@
 ACCOUNT_EMAIL_CONFIRMATION_AUTHENTICATED_REDIRECT_URL = 'user/'
 ACCOUNT_EMAIL_VERIFICATION = 'mandatory'
 ACCOUNT_EMAIL_REQUIRED = True
+
+# Celery configurations
+
+CELERY_BROKER_URL = 'redis://localhost:6379'
+CELERY_RESULT_BACKEND = 'redis://localhost:6379'
```
Comment (MathiasLantean) [inline target/settings.py:182]: It seems that these should be taken from an environment variable.

**Pull request merged**